source/includes/api/performance/_reward_points.md

```diff
@@ -89,6 +89,79 @@ Attribute | Type | Info
 page | `integer` |  Optional. The requested page number. Defaults to page 1 if not supplied.
 
 
+### Create Reward Points
+
+This endpoint creates a reward point for a specified promotion.
+
+> Request:
+
+``` http
+POST /api/v2/performance/promotions/:promotion_id/reward_points HTTP/1.1
+Authorization: Token token=xxx
+Content-Type: application/json
+
+{
+    "reference": "p1",
+    "user_group_ids": [355, 350],
+    "quantity": 2,
+    "points_per_quantity": 5,
+    "date_from": "2012-02-28",
+    "date_to": "2025-01-31",
+    "active": true
+}
+
+```
+
+
+> Response:
+
+``` http
+HTTP/1.1 201 CREATED
+Content-Type: application/json
+
+{
+    "id": 166,
+    "promotion_id": 4,
+    "name": "product 1",
+    "type": "product",
+    "reference": "p1",
+    "active": true,
+    "quantity": 2,
+    "points_per_quantity": 5,
+    "sale_date_from": "2012-02-28",
+    "sale_date_to": "2025-01-31",
+    "user_groups": [
+        350,
+        355
+    ],
+    "created_at": "2017-06-27T10:58:31.000+01:00",
+    "updated_at": "2017-06-27T10:58:31.000+01:00"
+}
+```
+
+
+#### HTTP Request
+
+`POST /api/v2/performance/promotions/:promotion_id/reward_points`
+
+#### URL Parameters
+
+Attribute | Type | Info
+--------- | ---- | ----
+promotion_id | `integer` |  Required. The ID of the promotion the reward point belongs to.
+
+#### Body Parameters
+
+Parameter | Type | Description
+--------- | ---- | -----------
+reference | string | Ref of the [performance product](#performance-module-performance-products) that the reward point is to be claimed against.
+user_group_ids | array | An array of all of the user groups that should be eligible for the reward point.
+quantity | integer | The quantity that must be claimed to qualify for the reward point.
+points_per_quantity | integer | The number of points to award per quantity.
+date_from | date | The date the reward point is eligible from.
+date_to | date | The date the reward point is eligible to.
+active | boolean | The active status of the reward point.
```

Comment:
Is it worth noting where these are mandatory/optional? Thinking specifically about user_group_ids being optional